### DISCUSSION TITLE
How to modify the default <nerdctl.toml> path

### DISCUSSION THREAD

**Discussion by imaxun** [containerd/nerdctl#1819] [Q&A] [answered]
### What is the problem you're trying to solve

How to modify the default <nerdctl.toml> path

### Describe the solution you'd like

How to modify the default <nerdctl.toml> path

**Comment by djdongjin** [comment, marked as answer]
`$NERDCTL_TOML` env variable:

https://github.com/containerd/nerdctl/blob/main/docs/config.md#file-path
